libr/bin/bfile.c

```diff
@@ -696,6 +701,12 @@ R_API void r_bin_file_free(void /*RBinFile*/ *bf_) {
 	free (a);
 }
 
+R_IPI RBinFile *r_bin_file_xtr_load_buffer(RBin *bin, RBinXtrPlugin *xtr, const char *filename, RBuffer *buf, ut64 file_sz, ut64 baseaddr, ut64 loadaddr, int idx, int fd, int rawstr) {
+	ut64 sz;
+	const ut8 *bytes = r_buf_data (buf, &sz);
```

Comment:
mmm, `r_buf_data` returns the *whole buffer*, so even if you do a `r_buf_seek` before it returns the entire buffer anyway (making the previous r_buf_seek useless)... Do you want to get only the bytes from the current seek to the end of the buffer?